packages/cubejs-schema-compiler/scaffolding/ScaffoldingTemplate.js

```diff
@@ -40,16 +40,16 @@ class ScaffoldingTemplate {
   schemaDescriptorForTable(tableSchema) {
     return {
       cube: tableSchema.cube,
-      sql: `SELECT * FROM ${tableSchema.schema}.${tableSchema.table}`, // TODO escape
+      sql: `SELECT * FROM "${tableSchema.schema}"."${tableSchema.table}"`, // TODO escape
```

Comment:
We can't use `"` here because not all backends support it. Need use escape identifier methods.

Reply:
under BigQuery.js the way you doing escape for the column name is as follows:

```
escapeColumnName(name) {
    return `"${name}"`;
  }
```

which is no different to the way it is done now other than it's inline.  Are you looking anything differently.

Reply:
Updated,  please check if it works.

Reply:
Yep. Right. For BigQuery it's `"`. On other hand MySQL supports only `\``.